### PR TITLE
Signup frontend

### PR DESCRIPTION
**Pull request by imisiaina**

### CODE REVIEW

File: frontend/src/components/app/App.css [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+}
+
+input, button {
+  border: none;
+  outline: none;
+  background: none;
+  font-family: 'Open Sans', Helvetica, Arial, sans-serif;
+}
+
+/* $contW: 900px;
```
Comment (tomallens) [inline frontend/src/components/app/App.css:19]: What's going on here in the comment?

File: frontend/src/components/app/App.css [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+  left: 0;
+  top: 50px;
+  width: 100%;
+  padding: 0 20px;
+  text-align: center;
+  color: #fff;
+  transition: transform 1.2s ease-in-out;
+}
+
+/* .img__text:before {
```
Comment (tomallens) [inline frontend/src/components/app/App.css:159]: and here

File: frontend/src/components/app/App.css [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+.sign-up {
+  transform: translate3d(-900px, 0, 0);
+}
+
+.cont.s--signup .sign-up { 
+  transform: translate3d(0, 0, 0);
+}
+
+/* .icon-link {
+  position: absolute;
```
Comment (tomallens) [inline frontend/src/components/app/App.css:332]: and here

**Review by Calumwi — APPROVED** [review]
I was trying to think of a Feta related pun but I thought it'd be too cheesy

**Pull request merged**